elasticapm/instrumentation/register.py

```diff
@@ -94,11 +94,21 @@
         ]
     )
 
+# These instrumentations should only be enabled if we're instrumenting via the
+# wrapper script, which calls register_wrapper() below.
+_wrapper_register = {
+    "elasticapm.instrumentation.packages.flask.FlaskInstrumentation",
+}
+
 
 def register(cls):
     _cls_register.add(cls)
 
 
+def register_wrapper():
```

Comment:
I would give this function another name, e.g. `register_wrapper_instrumentations`, to make it a bit clearer what it does.